updater/internal/action/service_action.go

```diff
@@ -38,6 +39,10 @@ func (s ServiceStopAction) Rollback() error {
 	return s.svc.Start()
 }
 
+func (s ServiceStopAction) String() string {
+	return "ServiceStopAction{}"
+}
```

Comment:
I don't think we need this if we're trying to satisfy the `Stringer` interface. Without this i'll likely print something like  `&{}`. This isn't super helpful if we need to know the type of the action from the string but if we don't really care I would omit this.

Reply:
I want to see what this looks like when manually testing; I've removed it for now.

Reply:
I'm going to keep these removed for the service actions, and keep it for the file action; It seems it prints the type, but not any of the contained information, which I think would be helpful for the CopyFileAction.